### ISSUE TITLE
handleRequest() not resulting in isSubmitted for DELETE method

### ISSUE THREAD

**Issue by dr-matt-smith** [sadikoff/crud-maker-bundle#6]
clicking a DELETE button in the Edit form doesn't result in 
isSubmitted() being true after a handleRequest()
so the actions to remove/flush the entity aren't executed, and no delete from DB takes place

to get this to work I had to add in a condition to submit the form, adopting example from [Symfony docs](https://symfony.com/doc/current/form/direct_submit.html)

but should this be necessary?

if required, then we need to update the Generated code for the Controller ...

```
    public function delete(LoggerInterface $logger, Request $request, Category $category)
    {
        $logger->info('1 :: got here');

        $form = $this->createDeleteForm($category);
        $form->handleRequest($request);

        $logger->info('2 :: isSubmitted = ' . $form->isSubmitted());

        if ($request->isMethod('DELETE')) {
            $form->submit($request->request->get($form->getName()));

            if ($form->isSubmitted() && $form->isValid()) {
                $logger->info('3 :: in the DELETE bit');
                $em = $this->getDoctrine()->getManager();
                $em->remove($category);
                $em->flush();
            }
        }
```

**Comment by sadikoff** [comment]
Is session and csrf enabled in your project?

**Comment by sadikoff** [comment]
@dr-matt-smith probably fixed in v1.0.3 can you please test new version?

**Comment by dr-matt-smith** [comment]
older version - was fine after requiring **security-csrf**

thanks .. matt ..
